app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -199,7 +201,7 @@ class AppVersionActivityTest {
   fun testAppVersionActivity_loadFragment_onBackPressed_displaysAdministratorControlsActivity() {
     ActivityScenario.launch<AdministratorControlsActivity>(
       launchAdministratorControlsActivityIntent(
```

Comment:
Change this function in line 233 to accept `internalProfileId` instead of `profileId`.
The final function will look something like this
```
  private fun launchAdministratorControlsActivityIntent(internalProfileId: Int): Intent {
    return AdministratorControlsActivity.createAdministratorControlsActivityIntent(
      ApplicationProvider.getApplicationContext(),
      internalProfileId
    )
  }
```

And then wherever this function is used, mention the named argument as `internalProfileId`.

Reply:
done